src/hotspot/os/linux/os_linux.cpp

```diff
@@ -2030,9 +2031,10 @@ void * os::Linux::dll_load_in_vmthread(const char *filename, char *ebuf,
 
   if (!_stack_is_executable) {
     for (JavaThreadIteratorWithHandle jtiwh; JavaThread *jt = jtiwh.next(); ) {
-      if (!jt->stack_guard_zone_unused() &&     // Stack not yet fully initialized
-          jt->stack_guards_enabled()) {         // No pending stack overflow exceptions
-        if (!os::guard_memory((char *)jt->stack_end(), jt->stack_guard_zone_size())) {
+      StackOverflow* sto = jt->stack_overflow_state();
```

Comment:
So why use `sto` here when you use `overflow_state` above?
The GitHub review UI makes this difference more obvious than webrev...

Update: In other source files, you use `overflow_state`.

Reply:
I'll change it to overflow_state.  I should be consistent.  Fixed.